hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1280,7 +1281,32 @@ public static <T extends Record, R extends GeneratedMessageV3> R pbjToProto(
         }
     }
 
-    public static com.hedera.hapi.node.base.Key asPbjKey(@NonNull final JKey jKey) {
+    /**
+     * Tries to convert a PBJ {@link Key} to a {@link JKey} unchecked (the only {@link HederaKey} implementation);
+     * returns an empty optional if the conversion succeeds, but the resulting {@link JKey} is not valid.
+     *
+     * @param pbjKey the PBJ {@link Key} to convert
+     * @return the converted {@link JKey} if valid, or an empty optional if invalid
+     * @throws IllegalStateException if the conversion fails
+     */
+    public static @NonNull Optional<HederaKey> fromPbjKeyUnchecked(@Nullable final Key pbjKey) {
```

Comment:
we can use `fromPbjKey` method that is existing already right ?

Reply:
this is just for testing because some of the key generated by seeded source are not valid

Reply:
I think same as above